### PR TITLE
Revive ability to read really old dolt_schema tables

### PR DESCRIPTION
**Pull request by macneale4**
We have several old databases which we don't touch but are public on DoltHub. These databases present an error to users with the remove of the migration ability in 1.42.0. The change will bring them back.

### CODE REVIEW

File: go/libraries/doltcore/sqle/schema_table_test.go [hunk 1]
```diff
@@ -45,10 +47,47 @@ func TestAncientSchemaTableError(t *testing.T) {
 	}), sql.Collation_Default, "")
 	require.NoError(t, err)
 
-	_, _, err = db.GetTableInsensitive(ctx, doltdb.SchemasTableName)
-	require.Error(t, err)
-	require.Contains(t, err.Error(), "cannot migrate dolt_schemas table from v0.19.1 or earlier")
+	sqlTbl, found, err := db.GetTableInsensitive(ctx, doltdb.SchemasTableName)
+	require.NoError(t, err)
```
Comment (fulghum) [inline go/libraries/doltcore/sqle/schema_table_test.go:51]: (minor) Probably worth updating the test name from `TestAncientSchemaTableError` to something like `TestAncientSchemaTableMigration` now that it doesn't test for an error any more.

**Review by fulghum — APPROVED** [review]
Looks good!

**Comment by coffeegoddd** [comment]
@macneale4 __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>ebfd472</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>ebfd472</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Comment by coffeegoddd** [comment]
@macneale4 __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>9f3acfe</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>9f3acfe</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Pull request merged**